src/stories/Box/components/BoxBody.tsx

```diff
@@ -0,0 +1,10 @@
+interface CardBodyProps {
```

Comment:
I assume we have a clear distinction between a card and box component now, but be sure to rename any instances of 'Card' to 'Box', like here